File: coding-test/java/Level_03/bj03_12.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

public class bj03_12 {
    public static void main(String[] args) throws Exception{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        String line;
        while((line = br.readLine()) != null){
            String[] arr = line.split(" ");
            int a = Integer.parseInt(arr[0]);
            int b = Integer.parseInt(arr[1]);
            bw.write((a+b) + "\n");
        }
        bw.flush();
    }
}

File: coding-test/python/Level_03/bj03_12.py
""" import sys
input = sys.stdin.readline

while True:
    line = input()
    if not line:   # EOF
        break

    a, b = map(int, line.split())
    print(a + b) """

import sys
input = sys.stdin.readline

while True:
    try:
        a, b = map(int, input().split())
        print(a + b)
    except:
        break
